qiskit_experiments/data_processing/nodes.py

```diff
@@ -58,11 +65,35 @@ def _process(
         Raises:
             DataProcessorError: If the axis is not an int.
         """
-        standard_error = np.std(datum, axis=self._axis) / np.sqrt(datum.shape[0])
+        standard_error = np.std(datum, axis=self._axis) / np.sqrt(datum.shape[self._axis])
 
         return np.average(datum, axis=self._axis), standard_error
 
 
+class Normalize(DataAction):
+    """Normalizes the data."""
+
+    def _format_data(self, datum: Any, error: Optional[Any] = None):
+        """Format the data into numpy arrays."""
+        datum = np.asarray(datum, dtype=float)
+
+        if error is not None:
+            error = np.asarray(error, dtype=float)
+
+        return datum, error
+
+    def _process(
+        self, datum: np.array, error: Optional[np.array] = None
+    ) -> Tuple[np.array, np.array]:
+        """Normalize the data to the interval [0, 1]."""
+        min_y, max_y = np.min(datum), np.max(datum)
+
+        if error is not None:
+            return (datum - min_y) / (max_y - min_y), error / (max_y - min_y)
+        else:
+            return (datum - min_y) / (max_y - min_y), None
```

Comment:
Do you think we call this normalization? I guess when we say normalization we just need to divide values by the absolute maximum value for rescaling, or subtracting the mean value to shift the mean to zero, not minimum value. If we have outlier you may get very huge positive value. At least you should use some lower percentile (but I prefer this is done in separate node).

Reply:
There are many ways to rescale and normalize data. See here: https://en.wikipedia.org/wiki/Feature_scaling. To be more precise on the terminology I renamed this node to `MinMaxNormalize` see here: 56f85b7